src/Sylius/Bundle/CoreBundle/MessageDispatcher/ResendShipmentConfirmationEmailDispatcher.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\MessageDispatcher;
+
+use Sylius\Bundle\CoreBundle\Message\Admin\ResendShipmentConfirmationEmail;
```

Comment:
I'm also not convinced of the need of `Admin` part in namespace. From the perspective of the Core, it does not matter, as it does not know about the admin anyway. Moreover, in theory it could just as well be used in a shop.